.github/workflows/bencher.yml

```diff
@@ -423,9 +423,7 @@ jobs:
       - uses: rui314/setup-mold@v1
         with:
           mold-version: ${{ env.MOLD_VERSION }}
-      - uses: actions-rs/audit-check@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I want to double-check the output of CI on this PR to see if we really need `token: ${{ secrets.GITHUB_TOKEN }}`.